pkgs/by-name/sq/squirreldisk/package.nix

```diff
@@ -38,6 +38,11 @@ let
     '';
     distPhase = "true";
     dontInstall = true;
+
+    patches = [
+      # Update field names to work with pdu v0.11.0
```

Comment:
BTW: the breaking change was in the 0.9.3 -> 0.10.0 bump

Reply:
Also, please include a link to the PR in the comment so that the status of the patch can be easily checked on updates.

Reply:
Are we talking about a link to https://github.com/adileo/squirreldisk/pull/47